docs/api-reference/orthographic-view.md

```diff
@@ -8,12 +8,9 @@ The [`OrthographicView`] class is a subclass of [View](/docs/api-reference/view.
 
 To render, `OrthographicView` needs to be used together with a `viewState` with the following parameters:
 
-* `eye` (`Number[3]`, optional) - The eye position in world coordinates. Default `[0, 0, 1]`.
-* `lookAt` (`Number[3]`, optional) - The position being looked at. Default `[0, 0, 0]`.
-* `up` (`Number[3]`, optional) - The up direction. Default `[0, 1, 0]`.
-* `offset` (`Number[2]`, optional) - The offset of the viewport. Default `[0, 1]`.
-* `zoom` (`Number`, optional) - The zoom level of the viewport. Default `1`.
-* `minZoom` (`Number`, optional) - The min zoom level of the viewport. Default `0`.
+* `offset` (`Number[2]`, optional) - The offset of the viewport, in screen pixels. Default `[0, 0]` (the coordinate origin is projected to the center of the viewport).
+* `zoom` (`Number`, optional) - The zoom level of the viewport. Default `0` (one unit distance maps to one pixel on screen).
```

Comment:
Shall we add an extra sentence like "larger zoom value lead to larger element size"?
(since we reversed the newScale / startScale ratio)